frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -102,6 +103,7 @@ const DELETE_PUBLIC_LINK = "metabase/entities/actions/DELETE_PUBLIC_LINK";
 const Actions = createEntity({
   name: "actions",
   nameOne: "action",
+  schema: ActionSchema,
```

Comment:
`createEntity` will create an empty schema by default, but I want to make sure we use the correct schema in our entities helper